tests/processes/BearRunningTest.py

```diff
@@ -253,13 +253,16 @@ def setUp(self):
         self.file_name_queue.put("invalid file")
         self.local_bear_list.append(LocalTestBear(self.settings,
                                                   self.message_queue))
-        self.local_bear_list.append("not a valid bear")
+        self.local_bear_list.append(SimpleGlobalBear({},
+                                                     self.settings,
+                                                     self.message_queue))
```

Comment:
@sils1297 , What is your suggestion on how to handle this, because the error message says
'''
send_msg(message_queue,
                 timeout,
                 LOG_LEVEL.WARNING,
                 "A given local bear ({}) is not valid. Leaving "
                 "it out...".format(bear_instance.__class__.__name__),
                 Constants.THIS_IS_A_BUG)
'''
which would return 'str' for the string which was previously used in the test, instead of the "not a valid bear" message.